src/sentry/api/validators/notifications.py

```diff
@@ -170,7 +169,7 @@ def validate(
     validate_organizations(organization_ids_to_look_up, user=user, team=team)
 
     return {
-        (provider, type, scope_type, scope_id, value)
```

Comment:
Instead of updating this tuple, you should just update the for loop in `update_settings_bulk`.